uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/request-modal/details/leave.html

```diff
@@ -5,13 +5,13 @@
       <div class="col-xs-8">
         <div class="btn-group btn-group-justified">
           <label
-            ng-disabled="$ctrl.isMode('view') || ($ctrl.isRole('manager') && !$ctrl.isLeaveType('sickness'))"
+            ng-disabled="$ctrl.isMode('view') || (!$ctrl.isNewRequest() && !$ctrl.isLeaveType('sickness'))"
```

Comment:
Manager check was not returning proper values before in case of a new request. So removed the same and modified the logic to keep the disabled state as before.